platform/authorized_api/apps/index.json

```diff
@@ -10,6 +10,11 @@
     }
   ],
   "definitions": {
+    "App_Id": {
```

Comment:
Looks like it's not used anywhere. I'm not sure if this is unnecessary or if I made an error during conflict resolution.

Reply:
yep, `App_Id` and `App` are unused